app/controllers/problems_controller.rb

```diff
@@ -61,7 +61,7 @@ def show
   end
 
   def create_issue
-    body = render_to_string "issue_trackers/issue", layout: false, formats: [:txt]
+    body = render_to_string "issue_trackers/issue", layout: false, formats: [:md]
```

Comment:
No, I imagine the same, so that action will send an extra parameter like `issue_format=md` , so we can decide which format we will render the issue body..
And when we render the format, we add this parameter based on the issue_tracker engine, which will provide a `body_format` method. or something like that.